### PR TITLE
Applicant fetch method

### PR DESCRIPTION
**Pull request by anuroop15**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
-yes

#### Which User story does this relate to?
- #146344381

#### Brief feature description
- This will update the Applicant cards group when user fills in Applicant details

#### Does this PR block/effect other stories?
-NA

#### Do any commands needs to be run in order to run the project with this PR?
-npm run dev

#### Code coverage percent diff as a result of this PR?
-js -> 63%

#### JS errors diff as a result of this PR?
- 79 errors

### CODE REVIEW

File: app/controllers/rfa/a01_controller.rb [hunk 1]
```diff
@@ -28,7 +28,19 @@ def edit
...
+    applicants = params[:applicants]
 
+    application_response = {}
+
+    applicants_result = []
+    applicants.each do |applicant|
+      applicants_result << rfa_applicant_helper.create(params[:id], applicant)
+    end
+
+    application_response[:applicants] = applicants_result
```
Comment (lauherkcambria) [inline app/controllers/rfa/a01_controller.rb]: i'm talking about variable application_response

Comment (ratneshraval) [inline app/controllers/rfa/a01_controller.rb]: This is not ready yet, We'll have to update this and finish fetch story.

File: test/javascript/forms/cardsMain_tests.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-
-  it('to have cards class', function () {
-    expect(cardTag.className).toBe('cards')
-  })
-
-  it('expect to have 6 componets', function () {
-    expect(cardTag.children.length).toEqual(6)
-  })
-})
+// import React from 'react'
```
Comment (ratneshraval) [inline test/javascript/forms/cardsMain_tests.js]: To get the branch in for everyone to follow. We will add tests in separate PR.

File: app/javascript/forms/nameCard.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+import {DropDownField} from '../common/dropDownField'
+
+export default class NameCard extends React.Component {
+  constructor (props) {
+    super(props)
+    this.addCard = this.addCard.bind(this)
+    this.removeCard = this.removeCard.bind(this)
+    this.handleNameChange = this.handleNameChange.bind(this)
+    let nameCardObj = this.props.nameFields
+    this.state = {
```
Comment (ratneshraval) [inline app/javascript/forms/nameCard.js]: Code is updated.

File: app/javascript/forms/nameCard.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        last_name: nameCardObj.last_name,
+        other_names: nameCardObj.other_names
+      },
+      legalType: [{
+        id: 2,
+        value: 'Legal'
+      }]
+    }
+  }
+  removeCard (indexValue) {
```
Comment (ratneshraval) [inline app/javascript/forms/nameCard.js]: Name card is the group holder for other-name cards. So this method will be here.

**Pull request merged**